src/lib/service/high5/wave/s3/index.ts

```diff
@@ -62,6 +63,23 @@ export class S3 extends Base {
         return resp.data;
     }
 
+    /**
+     * Get a list of available wildcards of a specific engine version from the S3 bucket
+     * @param engineUrl Public url of the engine
+     * @param engineVersion Version of the enigne
+     * @returns Wildcards.json list of available wildcards
+     */
+    async getWildcardsOfEngine(engineUrl: string, engineVersion: string): Promise<WildcardRegistry | undefined> {
+        try {
+            const resp = await this.axios.get<WildcardRegistry>(`${engineUrl.replace("/index.json", "")}/${engineVersion}/Wildcards.json`, {
```

Comment:
Can you do me a favor and lowercase the `Wildcards.json` here. Sev asked me to do it on s3 and for the following deployments (so in engine repo) and I forgot to do it here.